.prettierignore

```diff
@@ -30,21 +29,21 @@
 /services
 /test
 /third_party
-/tools
 /ui
 /utility
 /v8
 /vector_icons
 /win_build_output
 /.storybook/
 
+# Ignore external dependencies:
+tools/crates/vendor/
+
 # Ignore github .yml files:
 .github/**/*.yml
 
 # Ignored markdown files:
-.github/PULL_REQUEST_TEMPLATE.md
-CONTRIBUTING.md
-README.md
+*.md
```

Comment:
is this required? I'm fine with formatting .md files by prettier.

Reply:
don't have a strong opinion, will return it. We still have some unformatted `.md`, I will add them as exclusions.